### PR TITLE
Upgrade liquibase and fix include issue

### LINKED ISSUE #2901: Support includes used with liquibase
As ibis developer I'd like to have the frank-runner support liquibase (relative) includes so that it behaves the same as a non frank-runner environment.

Background:
In our application we use a couple liquibase files.
One which sets up the database structure (tables, indexes, etc.), which is intended to run on every environment.
One which fill the database with sample data, which is intended to run only locally and on dev environment.

To prevent duplicate code, we made the 2nd liquibase file include the first liquibase file and have all kinds of inserts below that.

This works when not using the frank-runner, but doesn't when using the frank-runner.
In the ibisconsole I'm getting an error 'LiquibaseMigrator liquibase failed to initialize'. In the application.log I'm getting a stacktrace which tells me the included file cannot be found.

Earlier I've contacted @jacodg about this issue:
'[dinsdag 16:45] de Groot, J.L. (Jaco)
Ik denk wat er aan de hand is dat de DatabaseChangelog_database-local.xml gelezen wordt met een F!F specifieke classloader (directory classloader waarschijnlijk in geval van draaien met Frank!Runner). Het is F!F code die deze file leest en deze classloader gebruikt. Vervolgens wordt Liquibase aangeroepen met de inhoud van deze file als input. Liquibase gaat aan de slag en gaat zelf op zoek naar de include file in het algemene classpath die geen benul heeft van een F!F specifieke directory classloader

[dinsdag 16:46] de Groot, J.L. (Jaco)
Dus simpel gezegd Liquibase zou bewust moeten worden gemaakt van deze speciale classloader

[dinsdag 16:46] de Groot, J.L. (Jaco)
Vast meerdere wegen naar Rome

[dinsdag 16:47] de Groot, J.L. (Jaco)
Maar is denk ik iets dat in het F!F aangepast moet worden, dus issue aanmaken is denk ik het beste, of je moet je zelf willen verdiepen in de F!F code

[dinsdag 16:48] de Groot, J.L. (Jaco)
Op WebSphere in DTAP zal het goed gaan omdat het in de "standaard" webapp classloader / classpath zit'

### PR DESCRIPTION
**Pull request by alisihab**
Closes #2901

### CODE REVIEW

**Review by gvanbrakel — APPROVED** [review]

**Comment by gvanbrakel** [comment]
It would be good to add a test for this

File: core/src/main/java/nl/nn/adapterframework/jdbc/migration/LiquibaseResourceAccessor.java [hunk 1]
```diff
@@ -69,8 +72,9 @@ public InputStreamList openStreams(String relativeTo, String streamPath) throws
 	public InputStream openStream(String relativeTo, String path) throws IOException {
 		if(path.equals(resource.getSystemId())) {
 			return resource.openStream();
+		} else if(classLoader != null) { // script may contain include(s)
+			return ((ClassLoaderBase)classLoader).getResource(path, false).openStream();
```
Comment (nielsm5) [inline core/src/main/java/nl/nn/adapterframework/jdbc/migration/LiquibaseResourceAccessor.java:76]: I'm heavily leaning towards using `ClassUtils.getResourceURL(resource, path).openStream();`
(You'll have to add the correct classloader/scopeProvider to the bytesresources)

Comment (gvanbrakel) [reply core/src/main/java/nl/nn/adapterframework/jdbc/migration/LiquibaseResourceAccessor.java:76]: I agree with Niels

File: core/src/main/java/nl/nn/adapterframework/jdbc/migration/DatabaseMigratorBase.java [hunk 2]
```diff
@@ -50,7 +50,7 @@ public abstract class DatabaseMigratorBase implements IConfigurationAware, Initi
 
 	protected Logger log = LogUtil.getLogger(this);
 	private @Setter IDataSourceFactory dataSourceFactory = null;
-	private Configuration configuration;
+	private @Getter Configuration configuration;
```
Comment (nielsm5) [inline core/src/main/java/nl/nn/adapterframework/jdbc/migration/DatabaseMigratorBase.java:53]: You can also let this class implement IScopeProvider :)

**Review by nielsm5 — APPROVED** [review]

**Pull request merged**